src/uu/unlink/src/unlink.rs

```diff
@@ -17,12 +17,13 @@ use libc::{lstat, stat, unlink};
 use libc::{S_IFLNK, S_IFMT, S_IFREG};
 use std::ffi::CString;
 use std::io::{Error, ErrorKind};
+use uucore::InvalidEncodingHandling;
 
 static NAME: &str = "unlink";
 static VERSION: &str = env!("CARGO_PKG_VERSION");
 
 pub fn uumain(args: impl uucore::Args) -> i32 {
-    let args = args.collect_str();
+    let args = args.collect_str(InvalidEncodingHandling::ConvertLossy).accept_any();
```

Comment:
rustfmt isn't happy here :)

Reply:
That's bad, must have gone wrong with the conflict resolution - i will fix it asap.

Reply:
np